crates/consensus-logic/src/l1_handler.rs

```diff
@@ -189,11 +191,14 @@ pub fn verify_proof(
             Risc0Verifier::verify_groth16(proof, vk.as_ref(), &public_params_raw)
         }
         RollupVerifyingKey::SP1VerifyingKey(vk) => {
-            SP1Verifier::verify_groth16(proof, vk.as_ref(), &public_params_raw)
+            SP1Verifier::verify_groth16_raw(proof, vk.as_ref(), &public_params_raw)
         }
     });
     match res {
-        Ok(Ok(())) => Ok(()),
+        Ok(Ok(())) => {
```

Comment:
```suggestion
        Ok(Ok(_)) => {
```
We don't care about the type of this nested OK right?
This might also catch bugs in the future.